torch/csrc/jit/passes/quantization.cpp

```diff
@@ -503,8 +503,56 @@ void InsertQuantDequantNodesForParam(
   }
 }
 
+void InsertQuantDequantNodesForParam(
+    script::Method& method,
+    const std::string& param_name,
+    const std::function<std::tuple<std::string, float, int>(float, float)>&
+        getQParamFunc,
+    at::ScalarType t) {
+  AT_ASSERT(getQParamFunc != nullptr);
```

Comment:
`AT_ASSERT` is deprecated, please use `TORCH_CEHCK`, please replace other `AT_ASSERT` in the pr as well.

Reply:
done